types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -94,6 +94,32 @@ declare namespace GoogleAppsScript {
      * Styles that can be set on a range using Range.setBorder(top, left, bottom, right, vertical, horizontal, color, style).
      */
     enum BorderStyle { DOTTED, DASHED, SOLID, SOLID_MEDIUM, SOLID_THICK, DOUBLE }
+    /**
+     * Represents an image to add to a cell. To add an image to a cell, you must create a new image
+     * value for the image using SpreadsheetApp.newCellImage() and CellImageBuilder. Then you can
+     * use Range.setValue(value) or Range.setValues(values) to add the image value to the cell.
+     */
+    interface CellImage {
+      getAltTextDescription(): string;
+      getAltTextTitle(): string;
+      getContentUrl(): string;
+      getUrl(): string;
```

Comment:
```suggestion
      getUrl(): string | null;
```

> Gets the image's source URL; returns null if the URL is unavailable. If the image was inserted by URL using an API, this method returns the URL provided during image insertion.
https://developers.google.com/apps-script/reference/spreadsheet/cell-image#getUrl()